main.go

```diff
@@ -266,6 +274,27 @@ func main() {
 		os.Exit(0)
 	}
 
+	// IN-OUT trigger
```

Comment:
Changed so that `transformer` is called provided the generator flag is used with the value `service.yml` and if there is also an already existing `docker-compose.yml` file.